dashboard/config/routes.rb

```diff
@@ -532,6 +532,14 @@
     post 'post_course_survey/submit', to: 'post_course_survey#submit'
     get 'post_course_survey/:course_initials', to: 'post_course_survey#new'
 
+    # Academic year workshops
+    get '/:workshop_subject/pre/(*agenda)', to: 'workshop_daily_survey#new_ayw_pre',
```

Comment:
What does the asterisk do? For using regex? Is `\d` not regex in the next route?

Reply:
this lets you define a variable for the [rest of the route](https://guides.rubyonrails.org/routing.html#route-globbing-and-wildcard-segments)--since we could have /AYW1/pre or /AYW1/pre/module/1, and we want module/1 as one variable. Also helpful for post where we can have /in_person and /module/1 and we want both of those represented as the same variable. Turns out rails routes can be quite flexible (I learned for this feature :) )

Reply:
TIL! Thanks!